polaris-react/src/components/Filters/Filters.scss

```diff
@@ -106,22 +106,30 @@
 
   &:hover,
   &:focus {
-    background: var(--p-color-bg-hover);
+    background: transparent;
     border-color: var(--p-color-border-hover);
 
     path {
       fill: var(--p-color-icon-hover);
     }
   }
 
+  &:hover {
+    border-style: solid;
+  }
+
+  &:focus {
+    outline-offset: var(--p-border-width-2);
+  }
+
   &:active {
-    background: var(--p-color-bg-active);
+    background: var(--p-color-bg-secondary-experimental);
     border-color: var(--p-color-border-hover);
   }
 
   &[aria-disabled='true'] {
-    background: var(--p-color-bg-disabled);
-    border-color: var(--p-color-border-disabled);
+    background: var(--p-color-bg-transparent-disabled-experimental);
+    border-color: transparent;
```

Comment:
No border in [Figma](https://www.figma.com/file/jLLkmo9r28hiqPvf4s90E4/Polaris-Gen-3-Components-%5Balpha%5D?node-id=44707%3A19512&mode=dev), setting to `transparent` to avoid layout shift.